DataDog/integrations-core
# [ci] setup libs in dd-agent now that JMX will be removed.

### What does this PR do?
Will pull agent libraries (JMX) now that they're not going to be living in the repo.

### Motivation

Once the jmxfetch JAR's are removed from the agent, we'll still need to download them for testing, etc. The tools have been added to `dd-agent` and the datadog-sdk-testing gem. So this just calls  those in the CI environments.

### Testing Guidelines

Once https://github.com/DataDog/dd-agent/pull/3220 and https://github.com/DataDog/datadog-sdk-testing/pull/60 (in that order) are merged and the new gem 0.7.0 is released - rerun this and make sure all tests are green.

### Additional Notes

This will remain broken until the steps above are completed, so DO NOT MERGE until CI is green (or we'll break `master` CI).
